src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountCommandController.cs

```diff
@@ -60,5 +61,20 @@ public async Task<IActionResult> AddEmployerVendorId([FromRoute] string hashedLe
             return NoContent();
         }
 
+        [HttpPatch("/legalentities/blockpayments")]
```

Comment:
Should be a noun.  Also either /blockedpayments or /legalentities/<id>/blockedpayments

Reply:
Updated to /blockedpayments as there are multiple IDs in the JSON payload